File: CS/ReportingExpressionFunctions/Form1.cs
using DevExpress.XtraReports.UI;
namespace ReportingExpressionFunctions
{
    public partial class Form1 : Form
    {
        public Form1()
        {
            InitializeComponent();
            Report1 report = new Report1();
            report.ShowRibbonPreviewDialog();
           
        }
    }
}

File: VB/ReportingExpressionFunctions/Form1.vb
Imports DevExpress.XtraReports.UI

Namespace ReportingExpressionFunctions

    Public Partial Class Form1
        Inherits Form

        Public Sub New()
            InitializeComponent()
            Dim report As Report1 = New Report1()
            report.ShowRibbonPreviewDialog()
        End Sub
    End Class
End Namespace
